documentation/quarkus/getting-started-quarkus.asciidoc

```diff
@@ -31,58 +31,56 @@ Follow the https://github.com/devonfw/ide/wiki/setup#setup[Setup] to install the
 .. Migrate a Spring app to Quarkus + 
-This guide shows, how to link:../guide-migration-spring-quarkus.asciidoc[migrate a Spring application to a Quarkus application with devon4j conventions]. 
+This guide shows how to link:../guide-migration-spring-quarkus.asciidoc[migrate a Spring application to a Quarkus application with devon4j conventions]. 
 
 
 == Create a REST service
 Now let's create our first REST CRUD service with Quarkus. 
-We give you the options to use a guide and start to code the service yourself,
-or just download a service that's ready to use. 
-
+We give you the options of using a guide to start to code the service yourself or to just download a service that's ready to use. 
 
 .Options
 . Create the service yourself +
-There is a good Quarkus guide for a simple https://quarkus.io/guides/rest-json#creating-your-first-json-rest-service[JSON REST service] that will guide you through your first application and will help you to implement defining endpoint with JAX-RS and an Entity that will be managed by the service and also how to https://quarkus.io/guides/rest-json#json[configure the JSON support].
+There is a good Quarkus guide for a simple https://quarkus.io/guides/rest-json#creating-your-first-json-rest-service[JSON REST service] that will guide you through your first application and help you with implement the definition of endpoints with JAX-RS and an Entity that will be managed by the service, and also how to https://quarkus.io/guides/rest-json#json[configure the JSON support].
 . Use an existing Quarkus project + 
-You don't want to code a service and just want to test some Quarkus functionalities? Just load a Quarkus https://github.com/quarkusio/quarkus-quickstarts#quick-start-list[sample project], provided for every existing quick start guide and the supported framework.
+You don't want to code a service and just want to test some Quarkus functionalities? Just load a Quarkus https://github.com/quarkusio/quarkus-quickstarts#quick-start-list[sample project] provided for every existing quickstart guide and the supported framework.
 Our Team also provides some Quarkus applications that are working and can be loaded and tested. 
-* https://github.com/devonfw-sample/devon4quarkus-reference[reference project] is a service that manages products. It contains the devon4j modern project structure, pagination, queries, a Postgres database, Jaeger tracing, Prometheus monitoring, SwaggerUI and support for Kubernetes deploy. 
+* https://github.com/devonfw-sample/devon4quarkus-reference[reference project] is a service that manages products. It contains the devon4j modern project structure, pagination, queries, a Postgres database, SwaggerUI, and support for Kubernetes deploy. To add OpenTelemetry support, see the following link:../opentelemetry-deployment.asciidoc[guide].
```

Comment:
The link is not working. This documentation is in the other repository. So use the full link here https://github.com/devonfw-sample/devon4quarkus-reference/blob/master/documentation/opentelemetry-deployment.asciidoc